joomla/joomla-cms
# [4.0] fixErrorInTemplateJS

Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/19318.

### Summary of Changes
I've corrected the assignment of the selected folder and added a minimized JS


### Testing Instructions
Go to Template - Customize, click "new file", choose file-type, enter name, choose directory where new file should be stored. (See https://github.com/joomla/joomla-cms/issues/19318)



### Expected result
Ability to choose a directory. 


### Actual result
Clicking a directory takes you back to the Template-Customize screen.
Clicking "create" will put the new file in the template's root.

After applying this patch the new file is saved in the selected directory.

### Documentation Changes Required
No
